sdk/textanalytics/Azure.AI.TextAnalytics/src/TextDocumentBatchStatistics.cs

```diff
@@ -16,12 +17,29 @@ namespace Azure.AI.TextAnalytics
     [CodeGenModel("RequestStatistics")]
     public partial class TextDocumentBatchStatistics
     {
-        internal TextDocumentBatchStatistics(int documentCount, int validDocumentCount, int invalidDocumentCount, long transactionCount)
+        internal TextDocumentBatchStatistics(
+            int documentCount,
+            int validDocumentCount,
+            int invalidDocumentCount,
+            long transactionCount)
+            : this(documentCount, validDocumentCount, invalidDocumentCount, transactionCount, default)
+        {
+        }
```

Comment:
Nit: given this is internal, is there any reason not to just change all the call sites to call the new constructor? This would result in slightly more performant code (fewer `.call` instructions, smaller stack) and less maintenance overall.